core/embed/rust/src/ui/model_tt/bootloader/confirm.rs

```diff
@@ -48,38 +47,39 @@ impl<'a> Confirm<'a> {
         icon: Option<Icon>,
         left: Button<&'static str>,
         right: Button<&'static str>,
-        confirm_left: bool,
-        confirm: (Option<&'static str>, Paragraphs<ParagraphVecShort<&'a str>>),
+        confirm: (
```

Comment:
agree about readability, but i kinda want to avoid having to split the `Confirm` into something like `ConfirmWipe` and `ConfirmInstall`, and if i just split the `confirm` argument `clippy` hates me for using 8 arguments. any ideas how to simplify here?

Reply:
I did not know about this `many arguments` limit of `clippy`. In that case, this approach is fine, I cannot suggest something quick and easy. It should be possible to somehow refactor and simplify the whole `Confirm` struct, but this is a `bootloader` code that will not be updated frequently, so IMO it is fine as is